Libraries/LibWeb/HTML/CookieStore.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2025, Johan Dahlin <jdahlin@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Bindings/CookieStorePrototype.h>
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/HTML/CookieStore.h>
+
+namespace Web::HTML {
+
+GC_DEFINE_ALLOCATOR(CookieStore);
+
+GC::Ref<CookieStore> CookieStore::create(JS::Realm& realm, GC::Ref<Page> page)
+{
+    return realm.create<CookieStore>(realm, page);
+}
+
+CookieStore::CookieStore(JS::Realm& realm, GC::Ref<Page> page)
+    : DOM::EventTarget(realm)
+    , m_page(page)
```

Comment:
I just went through the other merge request with the bulk of the code changes, having not read the full merge description until afterwards 😅 

I might drop some comments on that in a bit (it does look like a good start overall!), but for this commit - the convention in LibWeb is that each folder is it's own specification. Since this is in HTML folder, it indicates that CookieStore is part of HTML spec. But since CookieStore is it's own specification, I believe this should instead be in `LibWeb/CookieStoreAPI/CookieStore.cpp`

This will also mean that `CookieChangeEvent` will go into `CookieStoreAPI/CookieChangeEvent.cpp` /  `CookieStoreAPI/CookieChangeEvent.idl` like Tim mentioned. So all of those interfaces can also be removed from CookieStore.idl in this merge request.